02/app.js

```diff
@@ -1 +1,10 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const links = document.querySelectorAll('[data-url]');
+
+if(links !== null) {
```

Comment:
TO nigdy nie będzie `null` bo jest wiele elementów. Ewentualnie może to być pusta tablica ;)